client/driver/java.go

```diff
@@ -133,8 +133,17 @@ func (d *JavaDriver) Start(ctx *ExecContext, task *structs.Task) (DriverHandle,
 	// Get the environment variables.
 	envVars := TaskEnvironmentVariables(ctx, task)
 
+        // Look for jvm options
+        jvm_options, ok := task.Config["jvm_options"]
+        if !ok || jvm_options == "" {
+                d.logger.Printf("{DEBUG] no jvm options found")
```

Comment:
Why don't we make the debug message when we append the args since in a lot of cases people won't specify them.
Also small typo: { -> [